cmd/cmd_utils.go

```
@@ -543,3 +544,68 @@ func CheckForAtmosUpdateAndPrintMessage(atmosConfig schema.AtmosConfiguration) {
+// handleHelpRequest shows help content and exits only if the first argument is "help" or "--help" or "-h"
+func handleHelpRequest(cmd *cobra.Command, args []string) {
+	if (len(args) > 0 && args[0] == "help") || Contains(args, "--help") || Contains(args, "-h") {
+		cmd.Help()
+		os.Exit(0)
+	}
+}
+
+func showUsageAndExit(cmd *cobra.Command, args []string) {
+
+	var suggestions []string
+	var subCommand string = ""
```

golangci-lint (1.62.2)

ineffectual assignment to subCommand

(ineffassign)